src/functions/users/flextime/handler.ts

```diff
@@ -32,34 +22,57 @@ export const listUsersFlextimeHandler: APIGatewayProxyHandler = async (event) =>
     const usersWithFlextime = await Promise.allSettled(
       optedInUsers.map(async (severaUser) => {
         try {
+          /**
+           * TODO: This should be typed according to the spec response type for the lambda function
+           * Spec in the lambdas for the futur
+           */
           const flextime = await api.getFlextimeBySeveraUserId(severaUser.guid);
           return {
             user: {
               id: severaUser.guid,
               firstName: severaUser.firstName || "",
               lastName: severaUser.lastName || "",
-              email: severaUser.email || "",
-              attributes: {
-                severaUserId: severaUser.guid,
-                isActive: true
-              }
             },
             flextime: {
               totalFlextimeBalance: flextime?.totalFlextimeBalance || 0,
               monthFlextimeBalance: flextime?.monthFlextimeBalance || 0
             }
           };
         } catch (error) {
-          throw error;
+          /**
+           * Per-user error handling
+           */
+          console.error(`Error fetching flextime for user ${severaUser.guid}:`, error);
+          return {
+            user: {
+              id: severaUser.guid,
+              firstName: severaUser.firstName || "",
+              lastName: severaUser.lastName || "",
+              attributes: {
+                severaUserId: severaUser.guid,
+                isActive: true
+              }
+            },
+            flextime: null,
+            error: true,
+            /**
+             * Find the appropriate status code for the error
+             */
```

Comment:
This should be removed, we do not want to return the data like this from the catch as something has gone wrong (which is why we are in the catch). MIght be worth looking into what is typically returned from a catch if this is unclear.